curriculum/challenges/english/blocks/workshop-email-simulator/685d53b5e4f5784562a12d1b.md

```diff
@@ -9,7 +9,7 @@ dashedName: step-45
 
 Users should get confirmation when they successfully send an email. Let's improve the user experience by adding feedback to the `send_email` method.
 
-In the `send_email` method of the `User` class, add a print statement after the email is sent that shows confirmation. The message should include both the sender's name and the receiver's name in this format: `Email sent from [sender_name] to [receiver_name]!\n`
+The message should be `Email sent from [sender_name] to [receiver_name]!\n`, where `[sender_name]` is replaced by the sender's name and `[receiver_name]` is replaced by the receiver's name.
```

Comment:
you have deleted part of the instructions, now the `print` is not mentioned anymore. Please review the issue again